src/gam.py

```diff
@@ -10337,10 +10368,11 @@ def writeCSVfile(csvRows, titles, list_type, todrive):
 and follow recommend steps to authorize GAM for Drive access.''' % (admin_email)
       sys.exit(5)
     result = callGAPI(drive.about(), u'get', fields=u'maxImportSizes')
-    max_sheet_bytes = int(result[u'maxImportSizes'][u'application/vnd.google-apps.spreadsheet'])
-    if cell_count > 2000000 or columns > 256 or data_size > max_sheet_bytes:
+    if len(csvRows)*len(titles) > 2000000 or string_file.len > int(result[u'maxImportSizes'][MIMETYPE_GA_SPREADSHEET]):
```

Comment:
understanding what this line is doing is a lot harder without setting the above variables. Let's keep it as it was.

Reply:
Done